Web/HTML/html-structure/html-tags.md

```diff
@@ -11,11 +11,35 @@ category: must-know
 
 tags:
   - introduction
-
+---
+# Tags in HTML
 ---
 ## Content
 
-New content to go here. The author must be updated to match a valid Enki account.
+In HTML everything is written in tags.
+There is a start tag and an end tag and they differ by 1 character: "`/`".
+
+Tags are written as:
+```
+Start tag- <tag name>
+End tag- </tag name>
+```
+
+Some tags:
+- `<html>` (Defines an HTML document)
+- `<body>` (Documents body)
+- `<p>` (Paragraph)
+- `<br>` (Single line break)
+- `<!--Comment-->` (Inserts a comment)
+- `<!DOCTYPE>`
+
+
+In HTML5 only the `<!DOCTYPE>` tag is written in all caps.
+Other tags are not case sensitive. However, they are usually written in lowercase.
+This is because there is a wide range of developers who do not use HTML5, but rather XHTML. Because of that, they were influenced to write tags in lowercase so their HTML can be read by both markup languages.
+
+Also, there are elements which have no ending tag. These are called empty elements.
+
 
 ---
 ## Practice
```

Comment:
An element that has no closing tag is called "self-closing" - can you update the question to have the correct answer